mmv1/third_party/terraform/website/docs/r/dns_record_set.html.markdown

```diff
@@ -339,6 +339,97 @@ resource "google_compute_network" "prod" {
 }
 ```
 
+#### Primary-Backup with a Cross-region internal Application Load Balancer
```

Comment:
There are already a lot of examples on this page; is there any chance that this could be added to an existing example, and/or that any of them could be shortened?

Reply:
The bulk of this example (and the one above) is setting up all the components for the application load balancer. Is it fine to define only the forwarding rule and network (since they're the only direct dependencies) and leave off the rest?

Reply:
Now that I think about it, we could just make the forwarding rule and network data_sources instead of resources, right?

Reply:
In general, the goal of the examples in the [reference documentation](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/dns_record_set) is to show basic usage of the resource, not to cover every possible configuration. However, the examples that _are_ provided would ideally be complete / copy-pasteable and runnable.

Looking closer at this example, I'd lean towards not including it, since it's essentially identical to "Primary-Backup with a regional L7 ILB" (except for the `load_balancer_type` value, which should already be covered by the reference docs, and `region`, which it would make sense to not specify for a global resource.)

So I'd recommend:

- Delete this example (and possibly "Primary-Backup with a regional L7 ILB" as well) from the docs
- Update the docs for `load_balancer_type` to include `globalL7ilb` as a valid value

Reply:
Makes sense! I think it's fine to leave the original Primary-Backup example as is, and users can plug in their own forwarding rules as needed.